server/apps/immich/src/api-v1/system-config/response-dto/system-config-oauth-response.dto.ts

```diff
@@ -0,0 +1,9 @@
+export class SystemOAuthConfigResponseDto {
```

Comment:
Except that most fields are optional in the update dto and required in the response dto, so I think they're different signatures. Setting `clientId` is not required to be sent in an update, but that property will always exist on the response object.

Reply:
alright!

Reply:
@jrasm maybe it would be an option to have a `SystemConfigOAuthDto` with some fields optional, and then using `Required<SystemConfigOAuthDto>` where the fields are all required?